Assets/Scripts/Planets/PlanetInfo.cs

```diff
@@ -33,7 +33,7 @@ public class PlanetInfo : MonoBehaviour
     private const int TREEMAX = 20;
 
     [HideInInspector]
-    public int totalAnimals;
+    public int totalAnimals; //!!!!!!!
```

Comment:
Unsure how much value 7 exclamation points in a comment provide. Is this useful information?

Reply:
Sorry, it was just something to draw my attention towards when I was trying to figure out the remaining animals, but I never ended up using PlanetInfo and I forgot to remove it. It's removed now!